### PR TITLE
Description container

### LINKED ISSUE #196: Fix description container
The description container should have the following structure 
``` 
<DescriptionContainer>
{children}
< / DescriptionContainer>
``` 
where the children passed as props 

Ui fixes:
children should be located at the middle 
margin - 72 auto 114
Below 1280 - margin 72 24 114

### PR DESCRIPTION
**Pull request by Noamili**
TODO change the target branch to staging

close #196

https://user-images.githubusercontent.com/105313965/174303463-dcd71a6f-4d0d-4b28-a2ba-dd163a5a53e2.mov

https://user-images.githubusercontent.com/105313965/174303506-f2d1429c-c076-47e5-a263-7d9ec510f271.mov

### CODE REVIEW

File: components/layout/navbar/navbar.tsx [hunk 1]
```diff
@@ -101,6 +101,7 @@ const Navbar = ({ DesHeight }) => {
           align-items: center;
           box-shadow: 0px 16px 32px rgba(0, 0, 0, 0.25);
           width: 100%;
+          top: 0;
```
Comment (MichalPorag) [inline components/layout/navbar/navbar.tsx:104]: Why do you add the "top: 0"?

File: styles/Home.module.scss [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
-    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
-    grid-auto-rows: 1fr;
-    gap: 30px 45px;
-    justify-items: center;
-    padding: 0 25px;
-    margin: 0 auto;
-    max-width: 1769px;
-    margin-bottom: 100px;
-  }
+.cards__wrapper {
```
Comment (MichalPorag) [inline styles/Home.module.scss:3]: Please separate the changes of the grid to other PR.

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Great work! thank you so much :) 
Let's go over the CSS structure together in Zoom.

File: components/layout/navbar/navbar.tsx (hunk 1 above)
Comment (Noamili) [reply components/layout/navbar/navbar.tsx:104]: the navbar wasn’t sticking to the top of the screen but now I see it is acting fine so I will remove it

File: styles/Home.module.scss (hunk 2 above)
Comment (Noamili) [reply styles/Home.module.scss:3]: https://github.com/Pull-Request-Community/pull-request-community-website/pull/218 
done!

File: components/layout/navbar/navbar.tsx (hunk 1 above)
Comment (Noamili) [reply components/layout/navbar/navbar.tsx]: removed!

File: components/layout/layout.module.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  height: max-content;
+  display: flex;
+  flex-direction: column;
+  font-size: 1.6rem; // Based on 10px.
+  gap: 20px;
+  padding: 72px 0 114px;
+  margin: 0 auto;
+  max-width: 1280px;
+
+  @media only screen and (max-width: 768px) {
```
Comment (Noamili) [inline components/layout/layout.module.scss:11]: max width

File: components/layout/layout.module.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.description {
+  height: max-content;
+  display: flex;
+  flex-direction: column;
+  font-size: 1.6rem; // Based on 10px.
+  gap: 20px;
+  padding: 72px 0 114px;
+  margin: 0 auto;
+  max-width: 1280px;
```
[the remark this answers is not in the document]
Comment (Noamili) [reply components/layout/layout.module.scss:9]: width

[the remark this answers is not in the document]
Comment (Noamili) [reply components/layout/layout.module.scss:9]: add padding  top 136

File: components/layout/layout.module.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+  @media only screen and (max-width: 768px) {
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 72px 24px 114px;
```
Comment (Noamili) [inline components/layout/layout.module.scss:14]: add padding top 136

Comment (Noamili) [reply components/layout/layout.module.scss:14]: 80 px bottom

**Review by MichalPorag — APPROVED** [review]

**Comment by Noamili** [comment]
@MichalPorag ready for merge

**Review by MichalPorag — APPROVED** [review]
Geat work!

**Comment by MichalPorag** [comment]
@Noamili, I tried to fix the conflicts on the website, but it wasn't enough. Can you please fix them?

**Comment by Noamili** [comment]
> @Noamili, I tried to fix the conflicts on the website, but it wasn't enough. Can you please fix them?
can be merged!

**Pull request merged**